Error of missing header on mingw32

I am using mingw32 on linux to cross compile for Windows, and I get:

```
webview-win32.c:737:10: fatal error: webview2.h: No such file or directory
  737 | #include "webview2.h"
```

Renaming the include to `WebView2.h` fixes the issue, this happens because Linux file system is case sensitive.

--- a/webview-win32.c
+++ b/webview-win32.c
@@ -734,7 +734,7 @@ static int DisplayHTMLPage(struct webview *w) {
   return (-5);
 }
 
-#include "webview2.h"
+#include "WebView2.h"
 
 #if defined(__WebView2_LIBRARY_DEFINED__)
 #include "webview-win32-edge.c"

